conjure-core/src/main/java/com/palantir/conjure/defs/validator/ConjureDefinitionValidator.java

```diff
@@ -113,13 +112,19 @@ public void validate(ConjureDefinition definition) {
         }
 
         private static void verifyNameIsUnique(Set<TypeName> seenNames, TypeName name) {
-            boolean isNewName = seenNames.add(name);
-            Verify.verify(
-                    isNewName,
-                    "Type, error, and service names must be unique across locally defined and imported "
-                            + "types/errors: %s\n%s",
-                    seenNames,
-                    name);
```

Comment:
This is the only place we use `.verify` in the codebase; preferred using the same pattern to simplify the error handling.